app/user/rest.py

```diff
@@ -45,6 +45,10 @@
     check_throttle_for_requester,
 )
 from app.models import EMAIL_TYPE, SMS_TYPE, Permission
+from app.password_history.rest import (
+    add_password_for_user,
+    is_password_for_user_already_in_table,
```

Comment:
Personally I'd be tempted to call these "add_old_password_for_user" (or something similar) because we're not adding a users password (which could mean current password).  I'd also name the other function "has_user_already_used_password" as the name you've given it gives away how the history is implemented, whereas what we actually want to know is if they've already used it and we don't care how it's implemented under the hood.